packages/ui-react/src/components/FormField/FormField.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+import useOpaqueId from '@jwp/ott-hooks-react/src/useOpaqueId';
+
+import HelperText from '../HelperText/HelperText';
+import type { FieldProps } from '../../types/form-fields';
+
+import styles from './FormField.module.scss';
+
+type Props = {
+  children: (inputProps: { helperTextId: string; id: string }) => React.ReactNode;
+  showOptionalLabel?: boolean;
+  helperText?: React.ReactNode;
+} & FieldProps;
+
+export const FormField = ({ className, children, required, label, error, helperText, testId, name, editing = true, showOptionalLabel = true }: Props) => {
+  const { t } = useTranslation('common');
+  const formFieldClassName = classNames(styles.formField, className);
+  const id = useOpaqueId('text-field', name);
+  const helperTextId = useOpaqueId('helper_text', name);
+
+  return (
+    <div className={formFieldClassName} data-testid={testId}>
+      <label htmlFor={id} className={styles.label}>
+        {label}
+        {!required && editing && showOptionalLabel ? <span>{t('optional')}</span> : null}
```

Comment:
Do you think we need `showOptionalLabel` prop here? Can we just ignore the whole label tag based on the `label` prop absence?

Reply:
I didn't want to break the existing functionality of showing the optional label in the registration form when there is no label. But I don't like that it is needed. What do you think?